AppLibraryEnabler.xm

```diff
@@ -136,22 +132,6 @@ typedef struct SBHIconGridSize {
 	contentContainerView.frame = selfFrame;
 	searchResultsContainerView.frame = selfFrame;
 }
-- (void)_layoutSearchViews {
-	%orig;
-	MTMaterialView *searchBackdropView = [self valueForKey:@"_searchBackdropView"];
-
-	CGFloat width = [[UIScreen mainScreen] bounds].size.width;
-	CGFloat height = [[UIScreen mainScreen] bounds].size.height;
-
-	CGRect fullScreenFrame = CGRectMake(
```

Comment:
This part will be needed for application list (only height and top offset part) - without it overlay in this view is cut out on top and bottom.